.github/workflows/linting.yml

```
@@ -0,0 +1,26 @@
+name: Lint code
+on:
```

Propose we add a non-conditional value for push - it'll enable feedback on all changes for end-users to understand possible state of the linting changes they may be making prior to creating a PR, plus I'd anticipate comes at very little action minutes cost
```suggestion
on:
  push:
```